internal/metrics/metrics.go

```diff
@@ -18,12 +18,55 @@ const (
 )
 
 var (
-	RPCRequestsCounter = promauto.NewCounter(prometheus.CounterOpts{
-		Namespace: metricsNamespace,
-		Subsystem: metricsSubsystem,
-		Name:      "rpc_requests_total",
-		Help:      "Count of total RPC requests.",
-	})
+	rpcRequestsCounter = promauto.NewCounterVec(
+		prometheus.CounterOpts{
+			Namespace: metricsNamespace,
+			Subsystem: metricsSubsystem,
+			Name:      "rpc_requests_total",
+			Help:      "Count of total RPC requests.",
+		},
+		[]string{"code", "method"},
+	)
+
+	rpcRequestsDuration = promauto.NewHistogramVec(
```

Comment:
Were you planning on saving these for a future PR?

Reply:
The test in the PR description shows these metrics, so I'm assuming these automagically wired in somehow

Reply:
These are instrumented here: https://github.com/satsuma-xyz/node-gateway/pull/45/files#diff-6e56d7c057f6a96124b78f2b669dd7dface2718c304aece205017d114b3a7088R83